milli/src/search/facet/facet_distribution.rs

```diff
@@ -52,6 +72,7 @@ impl<'a> FacetDistribution<'a> {
             FacetType::Number => {
                 let mut key_buffer: Vec<_> = field_id.to_be_bytes().iter().copied().collect();
 
+                let distribution_prelength = distribution.len();
```

Comment:
In my point of view, it is only possible to have a non-empty `distribution` on String database, but this variable is used on Number database. 🤔
Is it a guard?